### PR TITLE
Update the pull request template

### PR DESCRIPTION
**Pull request by Dorukyum**
## Summary
Updates the PR template. I used it here so you can preview it. Open for ideas.

## Information

- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed).
- [x] This PR is **not** a code change (e.g. documentation, README, typehinting, examples, ...)

## Checklist

- [x] I have searched the open pull requests for duplicates.
- [ ] If code changes were made then they have been tested.
  - [ ] I have updated the documentation to reflect the changes.
- [ ] If `type: ignore` comments were used, a comment is also left explaining why.

### CODE REVIEW

**Comment by Dorukyum** [comment]
Idea 1: `I have searched the open pull requests for duplicates.` could be unindented and moven up.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -7,8 +7,14 @@
 <!-- Put an x inside [ ] to check it, like so: [x] -->
 
 - [ ] If code changes were made then they have been tested.
+    - [ ] I have searched the open pull requests for duplicates.
```
Comment (ArjunSharda) [inline .github/PULL_REQUEST_TEMPLATE.md:10]:
```suggestion
- [ ] I have searched open pull requests for duplicates.
```

**Review by ArjunSharda — CHANGES_REQUESTED** [review]

**Review by Lulalaby — APPROVED** [review]

**Review by ArjunSharda — APPROVED** [review]

**Review by Lulalaby — DISMISSED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -7,8 +7,14 @@
 <!-- Put an x inside [ ] to check it, like so: [x] -->
 
 - [ ] If code changes were made then they have been tested.
-    - [ ] I have updated the documentation to reflect the changes.
+- [ ] I have searched open pull requests for duplicates.
+- [ ] I have updated the documentation to reflect the changes.
```
[the remark this answers is not in the document]
Comment (Lulalaby) [reply .github/PULL_REQUEST_TEMPLATE.md:11]: Keep it like that

**Review by ArjunSharda — DISMISSED** [review]

**Comment by Dorukyum** [comment]
I think we should actually move "PR Information" above "Checklist" because it gives the main info about the pr and will affect the other checklist.

**Comment by Lulalaby** [comment]
true!

**Review by Lulalaby — DISMISSED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -2,14 +2,20 @@
...
 <!-- Put an x inside [ ] to check it, like so: [x] -->
 
-- [ ] If code changes were made then they have been tested.
-    - [ ] I have updated the documentation to reflect the changes.
-- [ ] If `type: ignore` comments were used, a comment is also left explaining why
 - [ ] This PR fixes an issue.
 - [ ] This PR adds something new (e.g. new method or parameters).
-- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
+- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed).
 - [ ] This PR is **not** a code change (e.g. documentation, README, typehinting, examples, ...)
```
Comment (ArjunSharda) [inline .github/PULL_REQUEST_TEMPLATE.md:12]:
```suggestion
[ ] This PR is **not** a code change (e.g. documentation, README, typehinting, examples, ...).
```

Comment (Lulalaby) [reply .github/PULL_REQUEST_TEMPLATE.md]: oh true

**Review by Lulalaby — APPROVED** [review]

**Review by Middledot — APPROVED** [review]

**Pull request merged**